app/main.py

```diff
@@ -1,36 +1,39 @@
 from typing import Union
+
 from fastapi import FastAPI
-from .models import Facetname
+
+from .models import Facetname, Hunger_game_filter
 
 app = FastAPI()
 
+
 @app.get("/")
 def hello():
     return {"message": "Hello from facets-knowledge-panels! Tip: open /docs for documentation"}
 
-@app.get("/brand/president")
-def ansewer_questions_brand_president():
+
+@app.get("/hunger-game-kp")
+def knowledge_panel_hunger_game(hunger_game_filter: Hunger_game_filter, value: Union[str, None] = None, country: Union[str, None] = None):
+    html = f"'<p><a href=\'https://hunger.openfoodfacts.org/?type={hunger_game_filter}&value_tag={value}&country={country}\'></a></p>\n'"
     return {
-            "knowledge_panels": [
-                {
-                    "hunger-game": {
-                        "type": "hunger-game",
-                        "level": "questions",
-                        "elements": [
-                            {
+        "knowledge_panels": [
+            {
+                "hunger-game": {
+                    "elements": [
+                        {
                             "element_type": "text",
                             "text_element": {
-                                "html":"<p><a href=\"https://hunger.openfoodfacts.org/?type=brand&value_tag=president\">Answer questions about brand president</a></p>\n"
+                                "html": str(html)
                             },
-                            },
-                        ],
-                    },
-                            
+                        },
+                    ],
                 },
-            ],
-        }
 
-@app.get("/{facet_name}/{facet_value}")
-def knowledge_panel(facet_name :Facetname,facet_value: str ):
-    return { "knowledge_panels": []}
+            },
+        ],
+    }
 
+
+@app.get("/{facet_name}/{facet_value}")
+def knowledge_panel(facet_name: Facetname, facet_value: str):
+    return {"knowledge_panels": []}
```

Comment:
the idea is more:

```python
panels = []
if facet_name in Hunger_game_filter:
  panels.append(hunger_game_kp(....))
return {"knowledge_panels": panels}
```

but I would better have the hunger_game_kp in a knowledge_panels.py